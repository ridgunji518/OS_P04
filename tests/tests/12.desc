Error Test 2: Hugealloc out of mem
