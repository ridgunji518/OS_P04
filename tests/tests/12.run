cd ../solution; ../tests/run-xv6-command.exp CPUS=4 Makefile.test test_12 | grep -E 'XV6_TEST_OUTPUT|XV6_TEST_ERROR'; rm *test_12* ;cd ../tests
