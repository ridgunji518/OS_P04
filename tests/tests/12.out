XV6_TEST_OUTPUT Max, 508MB - program base pages:0 huge pages:128
XV6_TEST_OUTPUT hugealloc - allocuvm out of memory
XV6_TEST_OUTPUT vmalloc fails correctly
